gma/src/include/firebase/gma/types.h

```diff
@@ -234,6 +235,45 @@ class AdError {
   AdErrorInternal* internal_;
 };
 
+/// Information about an ad response.
+class ResponseInfo {
```

Comment:
Moved this class up higher so it can be defined before it's used as a member of AdResult.